### PR TITLE
51 site config

### PR DESCRIPTION
**Pull request by HedgehogHolly**
Part of #51 

Changed title, email and stories link

and added git ignore dist

### CODE REVIEW

File: src/Views/Config.elm [hunk 1]
```diff
@@ -3,12 +3,12 @@ module Views.Config exposing (appTitle, email, helpline, helplineDisplay)
...
 appTitle : String
 appTitle =
-    "Pics, stories & info"
+    "Contemplating Action"
 
 
 email : String
 email =
-    "katja@neontribe.co.uk"
+    "ca@neontribe.co.uk"
```
Comment (katjam) [inline src/Views/Config.elm:11]: Did we agree this? Or going with contemplatingaction@neontribe.co.uk until we get a domain set up?

Comment (katjam) [reply src/Views/Config.elm:11]: Did we set up the group yet?

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,4 @@ elm-stuff
 # elm-repl generated files
 repl-temp-*
 node_modules
+dist
```
Comment (katjam) [inline .gitignore:6]: nice! You should not need to build - so you can also delete that dir if you want.

File: src/Views/Nav.elm [hunk 3]
```diff
@@ -23,7 +23,7 @@ view _ =
                 ]
             ]
         , nav []
-            [ navItem "stories" "#/stories" "story" "view-list" "Stories" "Stories"
+            [ navItem "question-circle" "#/stories" "find-out-more" "view-list" "Find Out More" "Find Out More"
```
Comment (katjam) [inline src/Views/Nav.elm:26]: Does it fit?

**Review by katjam — APPROVED** [review]
Couple of questions but I'm good for this to go in. :open_umbrella:

File: src/Views/Config.elm (hunk 1 above)
Comment (HedgehogHolly) [reply src/Views/Config.elm:11]: ca@neontribe.co.uk is what is currently in Kat's copy

File: .gitignore (hunk 2 above)
Comment (HedgehogHolly) [reply .gitignore:6]: Ed helped me with that as I was getting some errors before

File: src/Views/Nav.elm (hunk 3 above)
Comment (HedgehogHolly) [reply src/Views/Nav.elm:26]: It is a bit of a squeeze when it gets to an IPhone 5 size it cuts off the exit site text

Comment (katjam) [reply src/Views/Nav.elm:26]: OK Let's make it an acceptance criteria on the issue to address that. I think it'll be easier for @katquatermass to make judgements if we keep pushing stuff to staging.

File: .gitignore (hunk 2 above)
Comment (katjam) [reply .gitignore:6]: :) Teamwork!

File: src/Views/Config.elm (hunk 1 above)
Comment (katjam) [reply src/Views/Config.elm:11]: :+1: OK let's go with it. We can change.

**Pull request merged**